lms/djangoapps/organizations/tests.py

```diff
@@ -178,8 +180,6 @@ def test_organizations_detail_get(self):
         self.assertEqual(response.data['contact_name'], self.test_organization_contact_name)
         self.assertEqual(response.data['contact_email'], self.test_organization_contact_email)
         self.assertEqual(response.data['contact_phone'], self.test_organization_contact_phone)
-        self.assertIsNotNone(response.data['workgroups'])
-        self.assertIsNotNone(response.data['users'])
```

Comment:
Rather than removing these assertions, let's instead verify that they are no longer being returned for this particular API operation -- please include a comment explaining why.